sample.env

```diff
@@ -35,3 +35,9 @@ NODE_VERSION=16.15.1
 # Local direct setup
 API_HOST_NAME=http://localhost:6011
 REDIS_URL=redis://localhost:6379/0
+
+#############################################################
+
+# New Relic
+NEW_RELIC_CONFIG_FILE=newrelic.ini
+NEW_RELIC_LICENSE_KEY="don't write secrets to the sample file"
```

Comment:
These should be pulled from `cf env` along with the AWS creds, right? If so, let's note that in the instructions at the top of the file. (I can't suggest the change to that part of the file from this comment box, though, sorry)

Reply:
Yeah, that could be how it's retrieved. It also can just be left as-is if you're not trying to send data to new relic. I'll add all of that to the top of the file.